om-2.0.rdf

```diff
@@ -29292,6 +29327,40 @@
     <om:commonlyHasUnit rdf:resource="&om;teramole"/>
   </owl:Class>
 
+ <!-- Particle Concentration Unit Ontology -->
+
+  <owl:Class rdf:about="&om;ParticleConcentrationUnit">
+    <rdfs:label xml:lang="en">particle concentration unit</rdfs:label>
+    <rdfs:subClassOf rdf:resource="&om;Unit"/>
+    <owl:equivalentClass>
+      <owl:Class>
+        <owl:unionOf rdf:parseType="Collection">
+          <owl:Class>
+            <owl:oneOf rdf:parseType="Collection">
+              <om:UnitDivision rdf:about="&om;particleCountPerCubicCentimetre"/>
+            </owl:oneOf>
+          </owl:Class>
+        </owl:unionOf>
+      </owl:Class>
+    </owl:equivalentClass>
+  </owl:Class>
+
+  <owl:Class rdf:about="&om;ParticleConcentration">
+    <rdfs:subClassOf>
+      <owl:Restriction>
+        <owl:onProperty rdf:resource="&om;hasValue"/>
+        <owl:allValuesFrom>
+          <owl:Restriction>
+            <owl:onProperty rdf:resource="&om;hasUnit"/>
+            <owl:allValuesFrom>
+              <owl:Class rdf:about="&om;ParticleConcentrationUnit"/>
+            </owl:allValuesFrom>
+          </owl:Restriction>
+        </owl:allValuesFrom>
+      </owl:Restriction>
+    </rdfs:subClassOf>
+  </owl:Class>

```

Comment:
Again, please clarify in these two class definitions that we mean number concentration.

Reply:
Changed to Particle Number Concentration